recipes/imbalanced-learn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "imbalanced-learn" %}
+{% set version = "0.2.0.dev0" %}
+package:
+  name: imbalanced-learn
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: https://github.com/scikit-learn-contrib/{{ name }}.git
```

Comment:
Can you release a stable version first? Conda-forge is still discussing how to do pre-releases, release-candidate, etc.

Also, can you change the first lines to:

```yaml
{% set name = "imbalanced-learn" %}
{% set version = "0.2.0.dev0" %}

package:
  name: imbalanced-learn
  version: {{ version }}

source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
  sha256: ed97184e5665ea71b44222541b4c3d36daf4a8f6cf402df2976538082ff0509d
``` 

We prefer tarballs to `git_url` to avoid downloading more than the necessary and reduce the CI time.